# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

DetectedCar[] cars
DetectedPerson[] persons
bool external_observations
